templates/security/oval.html

```diff
@@ -0,0 +1,321 @@
+          </h3>
+          <div class="p-stepped-list__content">
+            <p>
+              Download the compressed XML:
+            </p>
+            <pre><code>wget https://security-metadata.canonical.com/oval/com.ubuntu.$(lsb_release -cs).cve.oval.xml.bz2</code></pre>
+            <p>
+              Uncompress the data:
+            </p>
+            <pre><code>bunzip2 com.ubuntu.$(lsb_release -cs).cve.oval.xml.bz2</code></pre>
+            <p>
+              Use OpenSCAP to evaluate the OVAL and generate an html report:
+            </p>
+            <pre><code>oscap oval eval --report report.htm com.ubuntu.$(lsb_release -cs).cve.oval.xml</code></pre>
+            <p>
+              The output is generated in the file report.htm, open it using your browser:
+            </p>
+            <pre><code>xdg-open report.htm</code></pre>
+            <p>
+              File naming convention:
+            </p>
+            <pre><code>com.ubuntu.releaseName.cve.oval.xml.bz2
+com.ubuntu.releaseName.usn.oval.xml.bz2</code></pre>
+            </div>
+          </li>
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">
+              Scanning an OCI Image
+            </h3>
+            <div class="p-stepped-list__content">
+              <p>
+                To scan an Ubuntu Official Cloud Image for known vulnerabilities, the manifest file and xml data can be used together. Unlike above where we were able to use the <code>lsb_release</code> command, you will need to manually enter the URL for the OVAL data.
+              </p>
+              <p>
+                In the example below we are using focal/20.04, you would replace 'focal' with the version you are inspecting.
+              </p>
+              <pre><code>wget https://security-metadata.canonical.com/oval/oci.com.ubuntu.focal.cve.oval.xml.bz2
+bunzip2 oci.com.ubuntu.focal.cve.oval.xml.bz2</code></pre>
+                <p>
+                  Download the manifest file for the image
+                </p>
+                <pre><code>wget -O manifest https://cloud-images.ubuntu.com/releases/focal/release/ubuntu-20.04-server-cloudimg-amd64-root.manifest</code></pre>
+                <p>
+                  Use OpenSCAP to evaluate the OVAL and generate an html report
+                </p>
+                <pre><code>oscap oval eval --report report.htm oci.com.ubuntu.focal.cve.oval.xml</code></pre>
+                <p>
+                  The output is generated in the file <code>report.htm</code>, open it using your browser
+                </p>
+                <pre><code>xdg-open report.htm</code></pre>
+                <p>
+                  File naming convention:
+                </p>
+                <pre><code>oci.com.ubuntu.releaseName.cve.oval.xml.bz2</code></pre>
+              </div>
+            </li>
+          </ol>
+        </div>
+        <div class="col-4 u-hide--small">
+          <p class="p-heading--two">&nbsp;</p>
```

Comment:
You should put the main heading off this session in its own row instead of this.

Reply:
the image applies only to step 1 though... or not sure what you mean.